vnext/package.json

```diff
@@ -21,20 +21,29 @@
     "validate-overrides": "react-native-platform-override validate"
   },
   "dependencies": {
-    "@babel/runtime": "^7.8.4",
+    "@babel/runtime": "^7.0.0",
+    "@react-native-community/cli": "^4.2.4",
+    "@react-native-community/cli-platform-android": "^4.2.3",
```

Comment:
Some of these are unneeded for Windows, but it's hard to reason about what's needed/not, and RN is already pulling all of these in, so we shouldn't need to download any additional packages.